recipes-support/rng-tools/rng-tools_%.bbappend

```diff
@@ -0,0 +1,2 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
```

Comment:
Can this go to OE-Core as well?

Reply:
perhaps yes but it should go along with the series to enable rv32, otherwise there will be questions as its purely rv32 patch